tools/atlas/packagesjson.nim

```diff
@@ -65,11 +65,57 @@ proc `$`*(pkg: Package): string =
   if pkg.web.len > 0:
     result &= "  website:     " & pkg.web & "\n"
 
+proc toTags(j: JsonNode): seq[string] =
+  result = @[]
+  if j.kind == JArray:
+    for elem in items j:
+      result.add elem.getStr("")
+
+proc singleGithubSearch(term: string): JsonNode =
+  # For example:
+  # https://api.github.com/search/repositories?q=weave+language:nim
+  var client = newHttpClient()
+  try:
+    let x = client.getContent("https://api.github.com/search/repositories?q=" & encodeUrl(term) & "+language:nim")
```

Comment:
```suggestion
    let x = client.getContent("https://api.github.com/search/repositories?language:nim&q=" & encodeUrl(term))
```

Less string ops.

Reply:
Lol, who cares, the HTTP request is 1000x more expensive and nothing of this is speed critical.